.circleci/config.yml

```diff
@@ -51,6 +51,28 @@ jobs:
       - run: *merge-check
       - run: python setup.py egg_info
 
+  pip:
+    docker:
+      - image: continuumio/miniconda3
+    steps:
+      - restore_cache:
+          keys: sample-data-v1
+
+      - checkout
+      - run: *skip-check
+      - run: *merge-check
+      - run: *apt-install
+      - run:
+          name: Install dependencies
+          command: pip install --progress-bar off -e .[all]
+      - run:
+          name: Run tests
+          command: PYTHONHASHSEED=42 pytest -m "not figure" -n 4
+
+    environment:
+          MPLBACKEND: agg
+          COLUMNS: 180
+
```

Comment:
Maybe this should be a cronjob in travis.

Reply:
yeah maybe